linera-views/src/common.rs

```diff
@@ -116,6 +141,9 @@ pub trait KeyValueStoreClient {
     /// Maximum number of simultaneous connections
     const MAX_CONNECTIONS: usize;
 
+    /// The maximal size of values that can be stored
```

Comment:
yes! Now we just need to decide if we want `.` at the end or not.

Reply:
Added the `.` if only for consistency.

Reply:
That's… a very pale dot. :smile: 
(It's not important, but it would be nice to be consistent.)